libvmi/accessors.c

```diff
@@ -141,6 +141,16 @@ vmi_get_offset(
     return vmi->os_interface->os_get_offset(vmi, offset_name);
 }
 
+status_t
+vmi_get_kernel_offset(
+    vmi_instance_t vmi, 
+    const char* symbol, 
+    const char* subsymbol, 
```

Comment:
I'm not particularly happy with this API. For internal use in LibVMI it's fine but it is really not intuitive to be publicly supported. Ie., when you don't specify a subsymbol it is looking for a kernel function / static variable, whereas with a subsymbol it looks for an offset in struct. These should really be two separate functions on the public side, as "vmi_get_kernel_offset" doesn't describe it accurately.